test/transform.js

```diff
@@ -43,31 +43,20 @@ test('transform adds sourcemap comment and uses cache on second time', function
 
       // Traceur converts all \s to /s so we need to do so also before comparing
       var fileConverted = file.replace(/\\/g, '/');
-      var sourceRootConverted = path.join(path.dirname(file), path.sep).replace(/\\/g, '/');
 
       t.deepEqual(
           sourceMap
         , { version: 3,
             file: fileConverted,
-            sources: [ fileConverted, '@traceur/generated/TemplateParser/1' ],
+            sources: [ path.basename(fileConverted) ],
             names: [],
             mappings: sourceMap.mappings,
-            sourceRoot: sourceRootConverted,
             sourcesContent: [
```

Comment:
So as far as I can tell this removes all our test coverage for source maps, which was the main sticking point in previous PRs :(. We can't break that...